apps/mull-ui/src/app/pages/messages/event-messages/event-message-list.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../../components';
+import EventBullet from '../../../components/event-bullet/event-bullet';
+import './event-message-list.scss';
+
+/* eslint-disable-next-line */
+export interface EventMessageListProps {}
+
+/*TODO: Remove this and replace with query of event message channels*/
+
+const messageChannels = [
+  {
+    eventTitle: 'Clean up of Rogers Park',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 1, 8, 20, 30, 1),
+  },
+  {
+    eventTitle: 'Lets do this guys!',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 5, 8, 20, 30, 1),
+  },
+  {
+    eventTitle: 'My god please clean this...',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 7, 8, 20, 30, 0),
+  },
+];
+
+export const EventMessageList = (props: EventMessageListProps) => {
+  const [searchString, setSearchString] = useState<string>('');
+
+  return (
+    <div className="event-messages-container">
+      <ChatHeader eventTitle="Event Messages" />
+      <div className="page-container">
```

Comment:
This div should be the first one

Reply:
This is what i did to switch the divs around to retain styling, lmk if theres a better way to do it 
![image](https://user-images.githubusercontent.com/35787279/109441833-5cc7fc00-7a04-11eb-9677-d5e2b3ca972d.png)